lib/file-finder.js

```diff
@@ -74,20 +77,27 @@ FileFinder.prototype.findInDirectoryOrParents = function(directory) {
     dirs = [];
     searched = 0;
     name = this.fileNames[0];
+    names = Array.isArray(name) ? name : [name];
 
-    while (directory !== child) {
-        dirs[searched++] = directory;
+    /* eslint-disable no-labels */
+    traversal:
```

Comment:
Ouch, labels. Is there any way to avoid this? Sorry, might be my personal prejudice against labels.